internal/tiltfile/wire.go

```diff
@@ -3,10 +3,13 @@ package tiltfile
 import (
 	"github.com/google/wire"
 
+	"github.com/windmilleng/tilt/internal/tiltfile/dockerprune"
+
 	"github.com/windmilleng/tilt/internal/tiltfile/k8scontext"
 )
 
 var WireSet = wire.NewSet(
 	ProvideTiltfileLoader,
 	k8scontext.NewExtension,
+	dockerprune.NewExtension,
```

Comment:
i'm surprised we're injecting the extension instead of just creating it inline? it's not like it has any special state we need to share or inject, so i'm not sure what you're getting from the additional complexity

Reply:
...nah, I was just following the k8sContextExt template w/o thinking. will fix.